examples/react-storybook-testing/.storybook/preview.tsx

```diff
@@ -0,0 +1,36 @@
+import { QueryClient, QueryClientProvider, setLogger } from "react-query";
```

Comment:
I think you should change to single quote instead of double quote.

Reply:
Thanks, `pnpm run lint` seems to ignore these files probably because the folder they live in starts with a `.`?